src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -485,7 +485,7 @@ private void CleanupRunspace(RunspaceDetails runspaceDetails)
         /// <returns>A RunspaceHandle instance that gives access to the session's runspace.</returns>
         public Task<RunspaceHandle> GetRunspaceHandleAsync()
         {
-            return this.GetRunspaceHandleImplAsync(CancellationToken.None, isReadLine: false);
+            return this.GetRunspaceHandleImplAsync(false, CancellationToken.None);
```

Comment:
Would still be helpful to name the boolean here

Reply:
I don't see how, there's one argument; hover over the function for its parameter name.

Reply:
> hover over the function for its parameter name.

Can't do that in GitHub. Ideally we wouldn't have boolean parameters, but labelling them definitely helps.

Reply:
Already fixed 😉 (Also GitHub is getting better at that! I think the Rich Code Nav is what enables it, and I need to get that setup for this repo)